modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -282,13 +287,43 @@ private void _validateSingleElementPath(
 		}
 	}
 
+	private boolean _validateUniquePathParameter(
+			long companyId, String pathParameter, long responseAPISchemaId)
+		throws Exception {
+
+		if (Objects.equals(pathParameter, "id") ||
```

Comment:
Is the `id` or `externalReferenceCode` reserved field names? if they are, ok; if not, maybe we should update this code